repo/packages/K/kubernetes/1/config.json

```diff
@@ -0,0 +1,205 @@
+          "type": "string"
+        },
+        "cpus": {
+          "default": 1.0,
+          "description": "CPU shares to allocate to each instance.",
+          "minimum": 0.0,
+          "type": "number"
+        },
+        "failover-timeout": {
+          "default": 300,
+          "description": "The failover_timeout for Mesos in seconds. If a new Kubernetes scheduler instance has not re-registered with Mesos this long after a failover, Mesos will shut down all running pods. Requires checkpointing to be enabled.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "apiserver-port": {
+          "default": 25502,
+          "description": "Read/write host port that the Kubernetes apiserver listens on.",
+          "minimum": 1,
+          "maximum": 65535,
+          "type": "integer"
+        },
+        "apiserver-secure-port": {
+          "default": 25500,
+          "description": "The Kubernetes apiserver port from which to serve HTTPS with authentication and authorization.",
+          "minimum": 1,
+          "maximum": 65535,
+          "type": "integer"
+        },
+        "scheduler-driver-port": {
+          "default": 25501,
+          "description": "Host port that the Kubernetes-Mesos scheduler driver listens on.",
+          "minimum": 1,
+          "maximum": 65535,
+          "type": "integer"
+        },
+        "scheduler-port": {
+          "default": 25504,
+          "description": "Host port that the Kubernetes scheduler listens on.",
+          "minimum": 1,
+          "maximum": 65535,
+          "type": "integer"
+        },
+        "controller-manager-port": {
+          "default": 25505,
+          "description": "Host port that the Kubernetes controller-manager listens on.",
+          "minimum": 1,
+          "maximum": 65535,
+          "type": "integer"
+        },
+        "instances": {
+          "default": 1,
+          "description": "Number of Kubernetes scheduler instances to run.",
+          "minimum": 0,
+          "maximum": 1,
+          "type": "integer"
+        },
+        "logging-verbosity": {
+          "default": 1,
+          "description": "Increase this value to obtain more detailed log messages.",
+          "type": "integer",
+          "minimum": 0
+        },
+        "mem": {
+          "default": 768.0,
+          "description": "Memory (MB) to allocate to each Kubernetes scheduler instance.",
+          "minimum": 128.0,
+          "type": "number"
+        },
+        "default-container-cpu-limit": {
+          "default": 0.25,
+          "description": "CPU shared to allocate to a containter in a pod which is not CPU limited.",
+          "minimum": 0.01,
+          "type": "number"
+        },
+        "default-container-mem-limit": {
```

Comment:
> I prefer something small in the sense of "fail early". Otherwise, users deploy apps and think they work, until they terminate when traffic raises.

Good point :+1: Fail-early + documentation seems like the best solution to this.